lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.ts

```diff
@@ -71,14 +85,19 @@ export class SearchChipAutocompleteInputComponent implements OnInit, OnDestroy {
         const value = (event.value || '').trim();
 
         if (value && this.isExists(value) && !this.isAdded(value)) {
-            this.selectedOptions.push(value);
+            if (this.allowOnlyPredefinedValues){
+                const index = this.autocompleteOptions.map(option => option.value.toLowerCase()).indexOf(value.toLowerCase());
```

Comment:
If you need to find specific element you can use `.find` instead of `.map`

Reply:
changed, thanks